curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da97c987a514959ada414.md

```diff
@@ -0,0 +1,165 @@
+---
+id: 641da97c987a514959ada414
+title: Step 23
+challengeType: 0
+dashedName: step-23
+---
+
+# --description--
+
+Now, call the `displayAuthors` function just like you did the other time.
+
+If you click the "Load More Authors" button after calling the function, it still won't work. That's because you are yet to add the click event listener to the button. you'll do that next.
```

Comment:
```suggestion
Now call the `displayAuthors` function with a portion of the author data just like you did before.

If you click the `Load More Authors` button after calling the function, it won't work. That's because you still have to add the `click` event listener to the button. You'll do that next.
```